maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -495,19 +495,6 @@ void logging(CliRequest cliRequest) {
         cliRequest.quiet = !cliRequest.verbose && commandLine.hasOption(CLIManager.QUIET);
         cliRequest.showErrors = cliRequest.verbose || commandLine.hasOption(CLIManager.ERRORS);
 
-        slf4jLoggerFactory = LoggerFactory.getILoggerFactory();
```

Comment:
Moving this code block is necessary because Jansi needs to be initialised before the execution of this block when `MessageUtils.setColorEnabled` is called ?

Reply:
Well, the MessageUtils.setColorEnabled must be called before calling slf4jConfiguration.setRootLoggerLevel witch load for the first time the MavenSimpleLogger class. Since this modified class use now static constants, its is necessary that the colour support is set right before loading the MavenSimpleLogger class.